homeassistant/components/glances/sensor.py

```diff
@@ -269,36 +269,38 @@ def _migrate_old_unique_ids(
         if sensor_type in ["fs", "sensors", "raid"]:
             for sensor_label, params in sensors.items():
                 for param in params:
-                    sensor_description = SENSOR_TYPES[(sensor_type, param)]
+                    sensor_description = SENSOR_TYPES.get((sensor_type, param))
+                    if sensor_description is not None:
+                        _migrate_old_unique_ids(
+                            hass,
+                            f"{coordinator.host}-{name} {sensor_label} {sensor_description.name_suffix}",
+                            f"{sensor_label}-{sensor_description.key}",
+                        )
+                        entities.append(
+                            GlancesSensor(
+                                coordinator,
+                                name,
+                                sensor_label,
+                                sensor_description,
+                            )
+                        )
+        else:
+            for sensor in sensors:
+                sensor_description = SENSOR_TYPES.get((sensor_type, sensor))
+                if sensor_description is not None:
```

Comment:
Can you use a walrus here?
If it's too long, you can rename `sensor_description` to just `description`.
```suggestion
                if sensor_description := SENSOR_TYPES.get((sensor_type, sensor)):
```